Content.Server/Glue/GlueSystem.cs

```diff
@@ -0,0 +1,56 @@
+using Content.Shared.IdentityManagement;
+using Content.Shared.Popups;
+using Content.Shared.Item;
+using Content.Shared.Interaction.Components;
+using Content.Shared.Glue;
+using Content.Server.Nutrition.EntitySystems;
+using Content.Shared.Interaction;
+using Content.Server.Nutrition.Components;
+
+namespace Content.Server.Glue
+{
+    public sealed class GlueSystem : EntitySystem
+    {
+        [Dependency] private readonly SharedAudioSystem _audio = default!;
+        [Dependency] private readonly SharedPopupSystem _popup = default!;
+        [Dependency] private readonly FoodSystem _food = default!;
+
+
+        public override void Initialize()
+        {
+            base.Initialize();
+
+            SubscribeLocalEvent<GlueComponent, AfterInteractEvent>(OnInteract);
+        }
+
+        // When glue bottle is used on item it will apply the glued and unremoveable components.
+        private void OnInteract(EntityUid uid, GlueComponent component, AfterInteractEvent args)
+        {
+            if (args.Handled)
+                return;
+
+            if (!args.CanReach || args.Target is not { Valid: true } target)
+                return;
+
+            if (!HasComp<ItemComponent>(target))
+                return;
+
+            if (!HasComp<GluedComponent>(target) || !HasComp<UnremoveableComponent>(target))
```

Comment:
If the item is already glued, the glue bottle just turns to trash with no feedback?

Reply:
nows checks if the bottle is already glued, if it is then it returns.